src/Symfony/Component/Validator/Resources/translations/validators.de.xlf

```diff
@@ -406,6 +406,26 @@
                 <source>The filename is too long. It should have {{ filename_max_length }} character or less.|The filename is too long. It should have {{ filename_max_length }} characters or less.</source>
                 <target>Der Dateiname ist zu lang. Er sollte nicht länger als {{ filename_max_length }} Zeichen sein.|Der Dateiname ist zu lang. Er sollte nicht länger als {{ filename_max_length }} Zeichen sein.</target>
             </trans-unit>
+            <trans-unit id="105">
+                <source>The password strength is too low. Please use a stronger password.</source>
+                <target>Das Passwort ist zu schwach.</target>
+            </trans-unit>
+            <trans-unit id="106">
+                <source>This value contains characters that are not allowed by the current restriction-level.</source>
+                <target>Der Wert enthält Zeichen, die nicht erlaubt sind.</target>
```

Comment:
It bothers me that the reference to the restriction level is lost. But I'm having a hard time finding a good translation for that term.

```suggestion
                <target>Der Wert enthält Zeichen, die auf der aktuellen Einschränkungsstufe nicht erlaubt sind.</target>
```

Reply:
I like your suggestion 👍